mne/preprocessing/ica.py

```diff
@@ -6,7 +6,7 @@
 
 from copy import deepcopy
 from inspect import getargspec, isfunction
-from collections import namedtuple, defaultdict
```

Comment:
... many people don't know it. Better rely on regular python dicts here for the API